.travis.yml

```diff
@@ -18,6 +18,8 @@ install:
   - pip install -U setuptools>=40.1.0
   - pip install -r requirements.dev.txt
   - pip install coveralls
+  # moto requires this package for EC2 mock, which is triggered even thought we aren't using it.
```

Comment:
Now that I have your explanation for this, I started feeling puzzled by this and did some research. I think this is a better fix for this problem: https://github.com/travis-ci/travis-ci/issues/7940#issuecomment-311411559

Reply:
Also note that you should document every non-obvious fix. Had you done that in the first place, we could have avoided yet another review round-trip. Just sayin' …